python/lsst/sconsUtils/state.py

```diff
@@ -449,17 +449,20 @@ def _saveState():
     if env.GetOption("clean"):
         return
 
-    import ConfigParser
+    try:
+        from configparser import ConfigParser
+    except ImportError:
+        from ConfigParser import ConfigParser
```

Comment:
These two bits are a py2/3 compatibility thing, no? Might be worth a comment to that effect so we can find it and cleanup when the "all-py3" days are upon us.